dev/manual_tests/card_collection.dart

```diff
@@ -324,7 +324,7 @@ class CardCollectionState extends State<CardCollection> {
         child: new SingleChildScrollView(
           child: new Container(
             height: cardModel.height,
-            decoration: new BoxDecoration(backgroundColor: theme.primaryColor),
+            color: theme.primaryColor,
```

Comment:
just a random thought, should we leave the 'background' part too? Wonder if we would ever add more features to Container like border etc where just 'color' could refer to foreground colors.